### DISCUSSION TITLE
Welcome to cluster.dev Discussions!

### DISCUSSION THREAD

**Discussion by voatsap** [shalb/cluster.dev#233] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kinseii** [comment]
My name is Petr Dondukov and I am a DevOps Engineer (https://d3vops.us). I want to improve our infrastructure built on terraform/terragrunt. As a result of investigations I found your project and was delighted with it, it is exactly how I envisioned IaC architecture. I want to move our IaC to cluster.dev.
